packages/scandipwa/src/component/Field/Field.component.js

```diff
@@ -193,21 +193,21 @@ export class Field extends PureComponent {
         const {
             type,
             setRef,
-            attr: { defaultChecked = false, ...newAttr } = {},
+            attr: { ...newAttr } = {},
```

Comment:
The use of the rest operator is irrelevant if you don't get other values first.
Remove the `newAttr` and use `attr` instead.

Reply:
this was there already, removed spread, renamed the variable on the fly

Reply:
Please don't rename variables without a reason. Use `attr` instead.